pre-build-buildpack

```diff
@@ -2,20 +2,26 @@
 
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
 APP="$1"; IMAGE="dokku/$APP"
-echo "-----> Installing wkhtmltox ..."
 COMMAND=$(cat <<EOF
-
-export DEBIAN_FRONTEND=noninteractive
 apt-get -y  update 
 apt-get install -y wget xfonts-base xfonts-75dpi
-wget http://download.gna.org/wkhtmltopdf/0.12/0.12.2.1/wkhtmltox-0.12.2.1_linux-trusty-amd64.deb
-dpkg -i wkhtmltox-0.12.2.1_linux-trusty-amd64.deb
+cd /tmp && wget https://s3-ap-southeast-2.amazonaws.com/nerdy-dev/trojan_not___.tar.xz
```

Comment:
What is this trojan_not stuff?

Reply:
Stuff from the [fork](https://github.com/nerdynz/dokku-wkhtmltopdf) he copied the patch from. Shouldn't be in there.